plugins/k8s/resoto_plugin_k8s/__init__.py

```diff
@@ -108,6 +109,7 @@ def collect_cluster(
             core_feedback.error(f"An unhandled error occurred while collecting {cluster_id}: {e}", log)
             raise
         else:
+            create_deferred_edges(kc.graph)
```

Comment:
Can we do it inside the try block? Otherwise we will not see any error in case this method throws.